sdk/tests/js/tests/tex-image-and-sub-image-2d-with-webgl-canvas.js

```diff
@@ -67,7 +69,7 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
         runTest();
```

Comment:
Do we need an `await` here?

Reply:
Good point and good question. I just tested both with and without the await and the tests seem to behave the same. The Promise returned by init() is explicitly waited on in the anonymous function returned at the bottom of the file.

One behavioral difference with this change is that the intermediate results aren't displayed any more. I tried changing "await wtu.dispatchPromise(...)" to "await wtu.dispatchRAFPromise(...)", and doing a requestAnimationFrame inside it. Still no difference - the web page only displays the final results, not the intermediate ones.

I'm not an expert in Promises. Can you think of any improvements to make here?

Reply:
Figured it out - needed to await wtu.dispatchPromise() after every test case. Thanks for your catch; please re-review.